pkg/users/password.jsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2020 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import cockpit from 'cockpit';
+import React from 'react';
+import { FormGroup, Progress, ProgressSize, ProgressMeasureLocation, TextInput } from '@patternfly/react-core';
+
+const _ = cockpit.gettext;
+
+export function password_quality(password, force) {
+    return new Promise((resolve, reject) => {
+        cockpit.spawn('/usr/bin/pwscore', { err: "message" })
+                .input(password)
+                .done(function(content) {
+                    var quality = parseInt(content, 10);
+                    if (quality === 0)
+                        reject(new Error(_("Password is too weak")));
```

Comment:
I cannot see this in the UI. If we decided not to show it (with which I am fine) then we don't need to do this, just resolve with 0 (or 1 if 0 would mean empty password).